In [2]:
import numpy as np

# Homework 1


## Mikko Karkkainen

<h3> <u> 1. Verify the limiting radius ratios of the 3-fold and 6-fold coordination </u> </h3>

<img src="https://dev-plebbit.s3.amazonaws.com/uploads/topic/picture/29/coordination_number_3.png" width=500px height=500px/>

<h4 align="center"> Figure 1. Visualisation of geometrical constraints of coordination number 5 </h4>

$$\cos(30^{\circ}) = \frac{R_A}{R_A+R_C} = \frac{\sqrt{3}}{2}$$

$$ R_A \Big( \frac{2}{\sqrt{3}}\Big) = R_C + R_A $$

 $$ R_C = R_A \Big( \frac{2}{\sqrt{3}} - 1\Big)  $$

 $$ \frac{R_C}{R_A} = \frac{2}{\sqrt{3}} - 1  $$

In [7]:
2/np.sqrt(3)-1

0.15470053837925168

<h3> Answer: $$ \frac{R_C}{R_A} = \frac{2}{\sqrt{3}} - 1 \approx 0.1547  $$  <h3>

<img src="https://dev-plebbit.s3.amazonaws.com/uploads/topic/picture/31/coordination_number_6.png" width=500px height=500px/>

<h4 align="center"> Figure 2. Visualisation of geometrical constraints of coordination number 4<h4>

$$ R_A^2 + R_A^2 = (R_A + R_C)^2 $$

$$ \sqrt{2} R_A = R_A + R_C $$

$$ R_C =  (\sqrt{2} - 1) R_A  $$ 

$$ \frac{R_C}{R_A} = \sqrt{2}-1  $$

In [29]:
np.sqrt(2)-1

0.41421356237309515

<h3> Answer: $$ \frac{R_C}{R_A} = \sqrt{2}-1 \approx 0.414 $$ </h3>

<h3> <u> 2.7 Density difference between SiO<sub>2</sub> and Al<sub>2</sub>O<sub>3</sub></u> </h3>

<em> Pauling's 1st rule : coordination number is determined by ratio between anion and cation </em>

<style> 
.container { width:100% !important; }
</style>

<table>
    <tr>
        <th>
        Ion
        </th>     
        <th>
        Radius for CN=4
        </th>   
        <th>
        Radius for CN=6
        </th>
    </tr>
    <tr>
        <td>
        Al<sup>+3</sup>
        </td>
        <td>
        0.39
        </td>
        <td>
        0.53
        </td>
    </tr>
    <tr>
        <td>
        Si<sup>+4</sup>
        </td>
        <td>
        0.26
        </td>
        <td>
        0.4
        </td>
    </tr>
</table>

Radius of O<sup>-2</sup> is approximately 1.4

In [38]:
print("ratio Al/O CN4: ", '{0:.2f}'.format(0.39/1.4))
print("ratio Al/O CN6: ", '{0:.2f}'.format(0.53/1.4))
print("ratio Si/O CN4: ", '{0:.2f}'.format(0.26/1.4))
print("ratio Si/O CN6: ", '{0:.2f}'.format(0.4/1.4))

ratio Al/O CN4:  0.28
ratio Al/O CN6:  0.38
ratio Si/O CN4:  0.19
ratio Si/O CN6:  0.29


Suprisingly, using data from the book, the radius ratio should be too small for 6-fold coordination for both cations. However, in reality, Al has a coordination number of 6 in Al<sub>2</sub>O<sub>3</sub> while Si has a coordination number of 4 in SiO<sub>2</sub>.

<em>Pauling's rule 2: total strength of bonds from each cation bonding with anion must be equal to anion charge.</em>

Aluminum: 6-fold coordination. Charge = 3+. Bond strength = 1/2
Silicon: 4-fold coordination. Charge = 4+. Bonds strength = 1.

Each oxygen can bond with 4 aluminum ions but only 2 silicon ions. The coordination numbers 6 for Al and 4 for O in Al<sub>2</sub>O<sub>3</sub> allow for a highly dense HCP structure, while 4 for Si and 2 for O in SiO<sub>2</sub> force a less dense tetrahedral chain.

<h3> <u> 2.13 Isomorphosity in garnets</u> </h3>

Garnet group has structural formula: X<sub>3</sub>Y<sub>2</sub>Z<sub>3</sub>O<sub>12</sub>

where CN for X = 8 and CN for Y = 6

Mg<sup>+2</sup> radius at CN=8 : could not find

Fe<sup>+2</sup> radius at CN=8 : could not find

Ca<sup>+2</sup> radius at CN=8 : 1.18

Mn<sup>+2</sup> radius at CN=8 : 1.1

At least for CN=6, Mg and Fe have similar radius and are thus interchangeable. However, Ca probably isn't, due to its larger size. This is connected to Pauling's 5th rule: differences in size of ions restricts number of different constituents.

According to a database, the Mg and Fe cations can be substituted by Mn. This is surprising, because according to the data, Mn has an ionic radius of 1.1, which would put it very close to the ionic radius of Ca+2, which it alas cannot substitute.

For Mg<sub>2</sub>SiO<sub>4</sub> and Fe<sub>2</sub>SiO<sub>4</sub> this analysis is easier, since the data for ionic radii is more available:

Mg<sup>+2</sup> radius at CN=6 : 0.72

Fe<sup>+2</sup> radius at CN=6 : 0.77

Mn<sup>+2</sup> radius at CN=6 : 0.67
    
Zn<sup>+2</sup> radius at CN=6 : 0.75
 
Ca<sup>+2</sup> radius at CN=6 : 1.00

Hg<sup>+2</sup> radius at CN=6 : 1.02

It appears that Mg and Fe cations could be substituted by Mn or Zn.

Ca cations may be substituted by Hg, since it shares the same charge and similar size.

<h3> <u> 2.14 Structure of unknown single crystal</u> </h3>

The empirical formula for the crystal was determined to be KF.AlF.BaO.MgO.Al<sub>2</sub>O<sub>3</sub>.5MgSiO<sub>3</sub>

The assignment hints that this crystal is similar to talc, pyrophillite and muscovite. What these three minerals have in common is that they are phyllosilicates. This means that they have a Si:O ratio of 2:5.

Phyllosilicates are characterised by the group Si<sub>4</sub>O<sub>10</sub>(OH)<sub>2</sub>, which is shared identically by almost all of its members. Muscovites have a small variation: One Si<sup>+4</sup> is substituted by Al<sup>+3</sup>, and the charge is balanced by a K<sup>+1</sup> cation. The (OH) group maybe substituted by F.

The closest form to a phyllosilicate which I was able to make from the given empirical formula was :

KAlF<sub>2</sub>.MgO.BaO.Al<sub>2</sub>O<sub>3</sub>.Mg<sub>5</sub>Si<sub>6</sub>O<sub>15</sub>

This way the charges are balanced. However, the Si:O ratio of the  entire crystal is not 2:5, which begs the question: is the crystal a phyllosilicate at all?

<h3> <u> 3.  Mineral Pairs </u> </h3>

<style> 
.container { width:100% !important; }
</style>

<table>
    <tr>
        <th>
        Pairs
        </th>     
        <th>
        Similarities
        </th>   
        <th>
        Differences
        </th>
    </tr>
    <tr>
        <td>
        Corundum(Al2O3) and Ilmenite(FeTiO3)
        </td>
        <td>
        Both have trigonal crystal structure. Ilmenite structure is derivative of Corundum structure.
        </td>
        <td>
        Corundum is hexagonal while Ilmenite is rhombohedral. Ilmenite has 2 different cations while corundum only has 1.
        </td>   
    </tr>
    <tr>
        <td>
        Albite (NaAlSi3O8) and Orthoclase (KAlSi3O8)
        </td>
        <td>
        Both are feldspar minerals. Specifically, they are both alkali feldspars.
        </td>
        <td>
        Unlike orthoclase, albite is an plagioclase feldspar as well as an alkali feldspar.
        </td>
    </tr>
    <tr>
        <td>
        High forms of Cristobalite and Tridymite
        </td>
        <td>
        Both are metastable in the temmperature range 270 C to 870 C
        </td>
        <td>
        High cristobalite is stable above 1470 C, Tridymite is stable from 870 to 1470. High Cristobalite is hexagonal, high Tridymite is cubic.
        </td>
    </tr>
    <tr>
        <td>
        Ilmenite and Perovskite
        </td>
        <td>
        Both have alternating cations. Both have one cation with CN = 6 (ideally). Both can take number of different crystal structures depending on the size and size difference of the cations.
        </td>
        <td>
        Ilmenite has coordination number 6 on both cations, perovskite has 12 on the bigger cation, 6 on the smaller. Perovskite is ideally cubic, while ilmenite is ideally rhombohedral.
        </td>
    </tr>
    <tr>
        <td>
        Calcium carbide and Titanium carbide
        </td>
        <td>
        Both are ionic carbides with rock-salt type crystal structure of alternating CN=6 anions and cations.
        </td>
        <td>
        Mechanical properties, TiC is very hard, while CaC is not.
        </td>
    </tr>
    <tr>
        <td>
        Kaolinite and Halloysite
        </td>
        <td>
        Both have same theoretical composition: Al2Si2O5(OH)4
        </td>
        <td>
        Unlike kaolinite, halloysite has water mono-layers between silicon sheets and a tubular structure.
        </td>
    </tr>
</table>

<h3> <u> 4.  Silicates and non-silicates </u> </h3>

<style> 
.container { width:100% !important; }
</style>

<table>
    <tr>
        <th>
        Mineral
        </th>     
        <th>
        Formula
        </th>   
        <th>
        Density
        </th>
    </tr>
    <tr>
        <td>
        Quartz
        </td>
        <td>
        SiO2
        </td>
        <td>
        2.66 g/cc
        </td>   
    </tr>
    <tr>
        <td>
        Cristobalite
        </td>
        <td>
        SiO2
        </td>
        <td>
        2.33 g/cc
        </td>
    </tr>
    <tr>
        <td>
        Orthoclase
        </td>
        <td>
        KAlSi3O8
        </td>
        <td>
        2.56 g/cc
        </td>
    </tr>
    <tr>
        <td>
        Corundum
        </td>
        <td>
        Al2O3
        </td>
        <td>
        4.02 g/cc
        </td>
    </tr>
    <tr>
        <td>
        Cordierite
        </td>
        <td>
        Mg2Al4Si5O18
        </td>
        <td>
        2.63 g/cc
        </td>
    </tr>
    <tr>
        <td>
        Periclase
        </td>
        <td>
        MgO
        </td>
        <td>
        3.93 g/cc
        </td>
    </tr>
    <tr>
        <td>
        Albite
        </td>
        <td>
        NaAlSi3O8
        </td>
        <td>
        2.6 g/cc
        </td>
    </tr>
    <tr>
        <td>
        Kaolinite
        </td>
        <td>
        Al2Si2O5(OH)4
        </td>
        <td>
        2.62 g/cc
        </td>
    </tr>
</table>

Silicates are unable to form close packed strutctures due to oxygen anions connecting to only 2 silicon cations and silicon cation connecting to only 4 anions. This forces a chain or web of tetrahedral units, which is not a dense structure.

Corundum has a HCP structure and periclase has a FCC structure, which are optimally packed. This explains the diference in density